team.daangn.com/src/components/JobPostingJsonLd.tsx

```diff
@@ -58,6 +59,15 @@ const JobPostingJsonLd: React.FC<JobPostingJsonLdProps> = ({
   overrides,
   defer = false,
 }) => {
+  const employmentTypeMapping:Record<EmploymentType,string> = {
```

Comment:
```suggestion
  const employmentTypeMapping: Record<EmploymentType, string | undefined> = {
```

요거 레코드 타입만 바꾸면 밑에 비교는 필요가 없을거같아용